.circleci/config.yml

```diff
@@ -78,11 +78,10 @@ jobs:
 
   sonarqube:
     docker:
-      - image: checkstyle/jdk-11-groovy-git-mvn:11.0.13__3.0.9__2.25.1__3.6.3
+      - image: checkstyle/jdk-11-groovy-git-mvn:11.0.20.1__2.4.21__2.42.0__3.9.5
 
     steps:
       - checkout
-      - run: apt-get install -y jq
```

Comment:
We do have install at https://github.com/checkstyle/contribution/commit/f059eb492b3ac9d94eaae39c69b55e17c57f9da6#diff-b2fa088093d6886acdf3dc702a7200e08fbcb57f4df0aea745b2ad401ff15fc9R6

Reply:
`image-name: &custom_img "checkstyle/jdk-11-groovy-git-mvn:11.0.20.1__2.4.21__2.42.0__3.9.5"` is a line-length violation. should i alter it to allow of greater length or do some hack to accomodate it?

Reply:
Try to shorten line, try `cs_img` instead of custom_img

Reply:
Sonarqube fails when i use `run: apt-get install -y jq`

Reply:
Remove
https://github.com/checkstyle/checkstyle/blob/503b26a00b6669775e74a483050c9202487bb00a/.circleci/config.yml#L85

You already have jq in image , no need to install it again

Reply:
Explanation is at https://github.com/checkstyle/checkstyle/pull/14012#issuecomment-1813681382